test/integration/users_signup_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class UsersSignupTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+  test "invalid signup information" do
+    get signup_path
+    assert_no_difference 'User.count' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.